lib/src/components/NxFilterInput/__tests__/NxFilterInput.test.tsx

```diff
@@ -69,7 +69,7 @@ describe('NxFilterInput', function() {
 
   it('passes unknown props to the div element', function() {
     const props = { className: 'extra-class', id: 'some-id' },
-        componentDiv = shallowComponent(props).find('div');
+        componentDiv = shallowComponent(props).find('div.nx-filter-input');
```

Comment:
This find shouldn't even be necessary.
```suggestion
        componentDiv = shallowComponent(props);
```